spec/features/result_filters/funding_spec.rb

```diff
@@ -80,13 +85,44 @@
         end
       end
     end
-  end
 
-private
+    describe "QS parameters" do
+      before do
+        stub_results_page_request
```

Comment:
Not for this PR but `stub_results_page_request` is very specific and we might need to keep an eye on its usage.

Reply:
Yeah... definitely. I was just thinking of it as something for this bit of work really until we have a functional results page. It will break when we implement filtering on that page so we can address it then.